### PR TITLE
ISSUE-903: add crashme(expression) function for testing

### LINKED ISSUE #903: [test] SELECT crashme(sleep(1))
**Summary**

`SELECT crashme(sleep(1))` will give a panic and see what's the fuse-query going, this is useful for testing.

### PR DESCRIPTION
**Pull request by BohuTANG**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

Add `crashme(expr)` function for testing how the system reacts under panic:

```
SELECT * FROM (SELECT crashme(sleep(1)));
```

And we can make a crash for some subquery after `If` implemented:
```
SELECT if(number%3=1, crashme(1), number) FROM numbers_mt(10) ;
```

## Changelog

- Build/Testing/CI
- Not for changelog (changelog entry is not required)

## Related Issues

Fixes #903 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by BohuTANG** [comment]
Add crashme function?

File: fusequery/query/src/datasources/system/crashme_table.rs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+impl CrashMeTable {
+    pub fn create() -> Self {
+        CrashMeTable {
+            schema: DataSchemaRefExt::create(vec![DataField::new("crash", DataType::Int64, false)]),
+        }
+    }
+}
+
+#[async_trait::async_trait]
+impl Table for CrashMeTable {
```
Comment (sundy-li) [inline fusequery/query/src/datasources/system/crashme_table.rs:35]: Better name to CrashMyself ?

Comment (BohuTANG) [reply fusequery/query/src/datasources/system/crashme_table.rs]: Have changed the `system.crashme` table to  `crashme` function.
Hmm, `crashme` short than `crashmyself`, this function is for testing only

File: tests/suites/0_stateless/02_0008_function_crashme.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+SELECT 1;
+SELECT crashme(sleep(1));
```
Comment (zhang2014) [inline tests/suites/0_stateless/02_0008_function_crashme.sql:2]: maybe we should also add `SELECT 1` after `SELECT crashme(sleep(1))` ?

Comment (BohuTANG) [reply tests/suites/0_stateless/02_0008_function_crashme.sql:2]: Good point, but the mysql client seems not work in the fuse-test.

Comment (BohuTANG) [reply tests/suites/0_stateless/02_0008_function_crashme.sql:2]: For the query `SELECT * FROM (SELECT crashme(sleep(1)))`, the clients get 3 times ERROR 2013:
```
mysql> SELECT * FROM (SELECT crashme(sleep(1)));
ERROR 2013 (HY000): Lost connection to MySQL server during query
No connection. Trying to reconnect...
Connection id:    8
Current database: *** NONE ***

ERROR 2013 (HY000): Lost connection to MySQL server during query
No connection. Trying to reconnect...
Connection id:    8
Current database: *** NONE ***

ERROR 2013 (HY000): Lost connection to MySQL server during query

```

Comment (BohuTANG) [reply tests/suites/0_stateless/02_0008_function_crashme.sql:2]: Create an issue here: https://github.com/datafuselabs/datafuse/issues/919

**Comment by BohuTANG** [comment]
I merge this patch for testing first.

**Pull request merged**